frontend/src/lib/constants.tsx

```diff
@@ -207,6 +207,7 @@ export const FEATURE_FLAGS = {
     HOG: 'hog', // owner: @mariusandra
     PERSONLESS_EVENTS_NOT_SUPPORTED: 'personless-events-not-supported', // owner: @raquelmsmith
     ALERTS: 'alerts', // owner: github.com/nikitaevg
+    ERROR_TRACKING: 'error-tracking', // owner: #team-replay
```

Comment:
Do we have an actual name or should we go with this for now?

Reply:
nice and simple 👍